listenbrainz/db/__init__.py

```diff
@@ -63,3 +63,12 @@ def run_sql_script_without_transaction(sql_file_path):
             connection.close()
         return True
 
+
+def create_test_database_connect_strings():
+    return ("postgresql://listenbrainz:listenbrainz@lb_db:5432/listenbrainz", "postgresql://postgres:postgres@lb_db/postgres",
```

Comment:
FWIW, I still think we should use a separate database name if at all possible, i.e.:
```suggestion
    return ("postgresql://listenbrainz:listenbrainz@lb_db:5432/listenbrainz_test", "postgresql://postgres:postgres@lb_db/postgres",
```

Reply:
Oh, good idea. Let me try that.

Reply:
Implemented